models/index.ts

```diff
@@ -61,7 +61,18 @@ export interface chooseBest {
   mode: 'PREPEND_OPTION' | 'APPEND_OPTION';
 }
 
-export type cohereParameters = generate | embed | chooseBest;
+export interface classify {
+  /** An optional string representing what you'd like the model to do. */
+  taskDescription?: string;
```

Comment:
nitpick: Should this value be moved to the end of the interface with outputIndicator since its optional so the required values are at the top?

Reply:
good catch, I like that pattern too. Fixed